README.md

```diff
@@ -57,7 +75,7 @@ of doing this:
    introduce the driver using the following snippet of code:
 
 	try {
-	  Class.forName("org.postgresql.Driver");
+	  Class.forName("legacy.org.postgresql.Driver");
```

Comment:
Prefixed all the package name with `legacy`. This will ensure there's absolutely no chance of package overlap (comparing to use suffix which could still potentially overlap).